packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -124,6 +124,41 @@ QUnit.module('rendering', () => {
         }
     });
 
+    QUnit.test('multiview items.visible option should always be default to true', function(assert) {
```

Comment:
[naming]
The test name described not what u're checking here. U say "option should be true" but checks that some class is presented.
```suggestion
    QUnit.test('item should be visible if no item.visible property is specified', function(assert) {
```

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)